lib/px/pxconfig.lua

```diff
@@ -55,6 +56,11 @@ _M.px_port = 443
 _M.ssl_enabled = true
 _M.enable_server_calls = true
 _M.send_page_requested_activity = true
+_M.base_url = _M.px_appId == 'PX_APP_ID' and 'sapi.perimeterx.net' or string.format('sapi-%s.perimeterx.net', _M.px_appId)
```

Comment:
I don't like this. It's not really readable and will confuse customers trying to input their app id. also, it looks like you now ask the customer to input app id twice